adhocracy4/comments/api.py

```diff
@@ -52,3 +53,29 @@ def destroy(self, request, *args, **kwargs):
         comment.save()
         serializer = self.get_serializer(comment)
         return Response(serializer.data)
+
+
+class CommentModerateSet(mixins.CreateModelMixin,
+                         mixins.RetrieveModelMixin,
+                         mixins.UpdateModelMixin,
+                         ContentTypeMixin,
+                         viewsets.GenericViewSet):
+
+    queryset = Comment.objects.all().order_by('-created')
+    serializer_class = CommentModerateSerializer
+    # permission_classes = (ViewSetRulesPermission,)
```

Comment:
do we not need this?

Reply:
Yes! Or not... You found the bit where there is still work to do! I need to make it work with the rules to be more specific. Currently everyone is allowed to edit the is_moderator_marked via the api.